clang/lib/Analysis/FlowSensitive/HTMLLogger.cpp

```diff
@@ -500,7 +500,7 @@ class HTMLLogger : public Logger {
     for (unsigned I = 0; I < CFG.getNumBlockIDs(); ++I) {
       std::string Name = blockID(I);
       // Rightwards arrow, vertical line
-      char ConvergenceMarker[] = u8"\\n\u2192\u007c";
+      char ConvergenceMarker[] = "\\n\u2192\u007c";
```

Comment:
@thevinster @martinboehme This change now fails on MSVC builds:

`warning C4566: character represented by universal-character-name '\u2192' cannot be represented in the current code page (1252)`

Maybe try this instead?
```c
const char *ConvergenceMarker = (const char*)u8"\\n\u2192\u007c";
```

Reply:
Thanks. Will revert and re-land with your suggestion. Sorry for the breakage!